packages/calypso-e2e/src/lib/components/support-component.ts

```diff
@@ -60,11 +37,7 @@ export class SupportComponent {
 			return;
 		}
 
-		// This Promise.all wrapper contains many calls due to a certain level of uncertainty
-		// when the support popover is launched.
 		await Promise.all( [
-			// Waits for all placeholder CSS elements to be removed from the DOM.
-			this.waitForQueryComplete(),
 			this.page.waitForSelector( selectors.supportPopoverButton() ),
 			this.page.click( selectors.supportPopoverButton() ),
 		] );
```

Comment:
I'm not sure I understand this chain - why do we need to wait for the same selector we're about to click here? I think it's what internally happens with `click()` - before it clicks, it waits for the given selector to be clickable, so the `click` itself should be sufficient.

Reply:
I think this was a mistake on my part. Can't think of reasons to explain this, all I can maybe come up with is that I mistyped a selector and forgot about it.